android-spaces-real-time-app/app/src/main/java/com/mux/realtimevideoexample/MainActivity.java

```diff
@@ -0,0 +1,153 @@
+package com.mux.realtimevideoexample;
+
+import android.Manifest;
+import android.content.pm.PackageManager;
+import android.os.Bundle;
+import android.view.ViewGroup;
+import android.widget.LinearLayout;
+import android.widget.Toast;
+
+import androidx.activity.result.ActivityResultLauncher;
+import androidx.activity.result.contract.ActivityResultContracts;
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.mux.sdk.webrtc.spaces.LocalParticipant;
+import com.mux.sdk.webrtc.spaces.MuxError;
+import com.mux.sdk.webrtc.spaces.Participant;
+import com.mux.sdk.webrtc.spaces.Space;
+import com.mux.sdk.webrtc.spaces.SpaceConfiguration;
+import com.mux.sdk.webrtc.spaces.Spaces;
+import com.mux.sdk.webrtc.spaces.Track;
+import com.mux.sdk.webrtc.spaces.views.TrackRendererSurfaceView;
+
+import java.util.HashMap;
+
+public class MainActivity extends AppCompatActivity {
+    private static final String[] PERMISSIONS = new String[] {
+        Manifest.permission.CAMERA,
+        Manifest.permission.RECORD_AUDIO
+    };
+
+    private final Space.Listener spaceListener = new Space.Listener() {
+        @Override
+        public void onJoined(Space space, LocalParticipant localParticipant) {
+            Toast.makeText(MainActivity.this, "Joined space "+space.getId()+" as "+localParticipant.getId(), Toast.LENGTH_SHORT).show();
```

Comment:
Nit: Let's make all these `LENGTH_LONG` (which in my opinion is still too short - I always miss these)